src/app/exercise/runner/runner.component.ts

```diff
@@ -235,15 +235,50 @@ function injectIframe(element: any, config: IframeConfig, runner: RunnerComponen
                 font-weight: 300;
                 background-color: #ffe;
               }
+              .mainerror {
+                color: black; 
+                font-size: larger;
+              }
+              .othererrors {
+                color: black; 
+                font-size: larger;
+              }
             `);
-            const diagnosticsHtml = diagnostics.map(diagnostic => `<li>Error in file <b>${diagnostic.file.fileName}</b>: 
-            ` + diagnostic.messageText + '</li>').join('');
-            setHtml(`
-<h2>Errors when compiling</h2>
- <div>Look in the editor for hints to fix it.</div>
- <ul>
-  ${diagnosticsHtml}
- </ul>`);
+            console.log(diagnostics);
+
+            const errList = diagnostics.map( (diagnostic, i) => {
+              if (i === 0) {
+                return `<li style="margin-bottom: 10px;"><div class="mainerror">Error in <b>${diagnostic.file.fileName}</b><br />` +
```

Comment:
You wouldn't need this if block to add the margin-bottom if you use CSS for the `ul`.

```css
ul.error-list li:first-child {
   margin-bottom: 10px;
}
```

```html 
<ul class="error-list">
   <li>This has margin bottom 10px</li>
   <li>This does not</li>
</ul>
```